Dockerfile

```diff
@@ -1,10 +1,9 @@
-FROM ubuntu:14.04
-MAINTAINER Wink
+FROM openjdk:8
 
-RUN apt-get update; apt-get install -y curl default-jdk
+RUN apt-get update; apt-get upgrade
```

Comment:
@kalmanchapman you sure you want to upgrade packages across the board on every build? This will make container builds non-deterministic. You could build something locally and have it run perfectly, and then the production container built in CI might fail b/c of package differences if something was updated between.